src/main/java/com/github/fge/jsonschema/main/JsonSchemaFactory.java

```diff
@@ -60,7 +60,7 @@
  *     <li>a {@link SyntaxValidator}, to validate schemas;</li>
  *     <li>a {@link JsonValidator}, to validate an instance against a schema;
  *     </li>
- *     <li>a {@link JsonSchema}, to validate instances against a fixed schema.
+ *     <li>a {@link JsonSchemaImpl}, to validate instances against a fixed schema.
```

Comment:
Yeah, not a big deal whatsoever.  Usually you just say the interface a factory returns so that clients don't have any expectations about the concrete type so you can change the implementation under the covers (like to one that uses Gson.  :)